core/arch/arm/plat-rcar/conf.mk

```diff
@@ -10,6 +10,12 @@ $(call force,CFG_SCIF,y)
 $(call force,CFG_CORE_LARGE_PHYS_ADDR,y)
 $(call force,CFG_CORE_ARM64_PA_BITS,36)
 
+# Disable core ASLR for two reasons:
+# 1. There is no source for ALSR seed, as Rcar platform
```

Comment:
`get_aslr_seed()` is a weak function which can be overridden by a platform provided version of that function. There's no strict dependency on DTB, it just happens to be a convenient way to pass the seed.